python/ray/util/client/server/server.py

```diff
@@ -510,6 +511,14 @@ def decode_options(
         return None
     opts = json.loads(options.json_options)
     assert isinstance(opts, dict)
+
+    if "placement_group" in opts:
+        pg_dict = opts["placement_group"]
+        if pg_dict is not None:
```

Comment:
It's possible to pass placement group `None` as an option.
I don't recall why, but one of the placement group tests does this.

Reply:
But we take it out on Line 314 right?

Reply:
Didn't see the relevant thing on line 314.
Could you link to the line that does that?

Perhaps I saw it give None because of an issue with client mode conversion logic?